samples/CloudClipboard/CloudClipboard/GarbageCollector/Program.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.Identity;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+
+namespace GarbageCollector
+{
+    public static class Program
+    {
+        private static readonly Uri BlobServiceUri = new Uri("https://cloudclips.blob.core.windows.net/");
+
+        public static async Task Main()
+        {
+            BlobClientOptions options = new BlobClientOptions();
+            options.Retry.MaxRetries = 10;
+            options.Retry.Delay = TimeSpan.FromSeconds(3);
+            options.Diagnostics.IsLoggingEnabled = true;
+
+
+            options.AddPolicy(new SimpleTracingPolicy(), HttpPipelinePosition.PerCall);
+
+
+            BlobServiceClient serviceClient = new BlobServiceClient(BlobServiceUri, new DefaultAzureCredential(), options);
+            await foreach (BlobContainerItem container in serviceClient.GetBlobContainersAsync())
+            {
+                BlobContainerClient containerClient = serviceClient.GetBlobContainerClient(container.Name);
+                await containerClient.DeleteAsync();
+            }
+        }
+    }
+
+    public class SimpleTracingPolicy : HttpPipelinePolicy
```

Comment:
By convention, this should likely be in its own file.

Reply:
This was done for the video.  I don't feel strongly, but would leave it so it matches what shows up on screen.

Reply:
Leave it at program.cs files